packages/eja/build.sh

```diff
@@ -4,5 +4,5 @@ TERMUX_PKG_LICENSE="GPL-3.0"
 TERMUX_PKG_VERSION=13.11.20
 TERMUX_PKG_SRCURL=https://github.com/eja/eja/archive/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=d19a8071d524373bcfceb5dec834546e2c66108593f0bf54ef984d8a0819e050
-TERMUX_PKG_EXTRA_MAKE_ARGS="DESTDIR=$TERMUX_PREFIX prefix=/"
+TERMUX_PKG_EXTRA_MAKE_ARGS="DESTDIR=${TERMUX_PREFIX} PREFIX=${TERMUX_BASE_DIR} prefix=/"
```

Comment:
This actually not a fix, `$TERMUX_BASE_DIR` can't be used here because system root (where bin, etc, share, var, tmp are available) is `$TERMUX_PREFIX`.

Paths defined in eja.lua will just be wrong:
```
 eja.path=_eja_path or '/'
 eja.pathBin=eja.pathBin or eja.path..'/usr/bin/'
 eja.pathEtc=eja.pathEtc or eja.path..'/etc/eja/'
 eja.pathLib=eja.pathLib or eja.path..'/usr/lib/eja/'
 eja.pathVar=eja.pathVar or eja.path..'/var/eja/'
 eja.pathTmp=eja.pathTmp or eja.path..'/tmp/'
 eja.pathLock=eja.pathLock or eja.path..'/var/lock/'
```

Reply:
Though, some of them will be wrong anyway, specifically `eja.pathBin` and `eja.pathLib`.

Reply:
exactly, the problem is that `TERMUX_PREFIX="${TERMUX_BASE_DIR}/usr"` but _eja_path is expecting a root directory, which TERMUX_BASE_DIR is indeed providing.

Reply:
TERMUX_BASE_DIR is not a root directory. It is for different purposes and normally shouldn't be used in package build scripts.

TERMUX_PREFIX is package installation root.

Reply:
ok, I see now where the problem was coming from. Thank you for clarifying it.